site/assets/site.css

```
@@ -0,0 +1,220 @@
+/*
+	Semantic palette for the publication, built on vendored reasonable-colors.css.
+	Components reference these aliases only, never --color-{name}-{shade} directly.
+
+	Mapping notes: Reasonable Colors has no warm-neutral/parchment hue, so --bg/--surface/--ink
+	trade the original site's sepia warmth for RC's cool gray scale — the nearest honest match,
+	not a forced one. --terra (accent, links) maps to amber; --verd (structural boxes, callouts)
+	maps to emerald; --bronze (kickers, headings) maps to cinnamon. Shade gaps of 3 hold body-text
+	contrast (4.5:1) and gaps of 4 hold AAA (7:1) per the RC contrast table.
+*/
+@import url("./reasonable-colors.css");
```

Stylelint (17.14.0)

[error] Expected "url("./reasonable-colors.css")" to be ""./reasonable-colors.css"" (import-notation)

(import-notation)